### ISSUE TITLE
Allow custom tmp path for Linux agent

### ISSUE THREAD

**Issue by georgebarnick** [amidaware/tacticalrmm#1017]
This may be an edge case but in situations where /tmp/ has locked down permissions or special handling on the system, Tactical RMM has trouble installing and executing scripts.

I've been able to verify this in one particular case, which is on CentOS 7 servers running a clean install of cPanel/WHM. From my understanding, cPanel locks the /tmp/ filesystem down using a script they call [securetmp](https://docs.cpanel.net/whm/scripts/the-securetmp-script/). Beyond that I haven't looked very much into this case and not entirely sure on to what extent it's locked down, but from what I can tell, execution is prevented entirely in that filesystem, even as root.

What I was able to confirm however is that during the install using the Linux installation script, it can't execute ./meshagent inside of its tmp directory it downloads to. Moving ./meshagent somewhere else (such as /home/) works and installs fine however. Once the agent is installed, if you try to execute a script, it gets Permission Denied inside of /tmp/trmmXXXXXXXXXX/. Example below:
```
Script Output
Last Run: Mar-22-2022 - 15:50
Run Time: 0.0005 seconds
Return Code: -1

Standard Error

fork/exec /tmp/trmm2651145699: permission denied
```

Since this may be an edge case and could vary depending on what types of Linux installs might handle /tmp/ differently, I was thinking it could be a good feature request to specify a custom temporary file path for Tactical RMM to use during the install process, rather than it being forced to use /tmp/.

**Comment by bbrendon** [comment]
I think this might be it. `noexec`
```
/dev/mapper/centos-tmp on /tmp type ext4 (rw,nosuid,noexec,relatime,seclabel,data=ordered)

```

**Comment by wh1te909** [comment]
I've pushed a fix to fallback to the current directory if creating it in the tmp dir fails

**Comment by wh1te909** [comment]
fixed in release 0.12.1 please check release notes. you'll need to manually update the agent if already installed since /tmp was used for agent update as well
